crypto/aes/aes_core.c

```diff
@@ -43,7 +43,989 @@
 #include <openssl/aes.h>
 #include "aes_local.h"
 
-#ifndef AES_ASM
+#if defined(OPENSSL_AES_CONST_TIME) && !defined(AES_ASM)
+typedef union
+{
```

Comment:
On previous line?

Reply:
I.e. `typedef union {`

Reply:
ah, I forgot the openssl style ... can do that.